.github/workflows/major-release.yml

```diff
@@ -0,0 +1,51 @@
+name: Major Release
+
+permissions:
+  issues: write
+  contents: write
+
+on:
+  schedule:
+    - cron: 0 0 15 * *
+
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check for release schedule
+        id: check-date
+        run: |
+          # Get the current month and day
+          MONTH=$(date +'%m')
+          DAY=$(date +'%d')
+          # We'll create the reminder issue two months prior the release
+          if [[ "$MONTH" == "02" || "$MONTH" == "08" ]] && [[ "$DAY" == "15" ]]; then
+            echo "create_issue=true" >> $GITHUB_ENV
+            release_date=$(date -d "$(date +%Y-%m-%d) +2 month" +"%d %B %Y")
+            echo "RELEASE_DATE=$release_date" >> $GITHUB_ENV
+            pr_max_date=$(date -d "$(date +%Y-%m-%d) +1 month" +"%d %B %Y")
+            echo "PR_MAX_DATE=$pr_max_date" >> $GITHUB_ENV
+          else
+            echo "create_issue=false" >> $GITHUB_ENV
+          fi
```

Comment:
Shouldn't we download the schedule from the Release repo instead of assuming the date hasn't changed?

Reply:
I think that could be an improvement yeah, but I think it's simpler as sending a PR to adjust the date in case of changes.